packages/vercel-edge/src/sdk.ts

```diff
@@ -182,10 +182,10 @@ export function setupOtel(client: VercelEdgeClient): void {
 }
 
 /**
- * Setup the OTEL logger to use our own logger.
+ * Setup the OTEL logger to use our own debug logger.
  */
 function setupOpenTelemetryLogger(): void {
-  const otelLogger = new Proxy(logger as typeof logger & { verbose: (typeof logger)['debug'] }, {
+  const otelLogger = new Proxy(debug as typeof debug & { verbose: (typeof debug)['log'] }, {
```

Comment:
hmm do we even need this? Can we not directly set e.g. `{ error: debug.error, warn: debug.warn, ... }` then here?

Reply:
(which imho is much nicer than the proxy anyhow)